tasks/copy_to_archive/schemas/output.json

```diff
@@ -1,10 +1,10 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/copy_to_glacier/schemas/output.json",
-  "title": "copy_to_glacier Lambda Output",
-  "description": "The output for the copy_to_glacier Lambda.",
+  "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/copy_to_archive/schemas/output.json",
+  "title": "copy_to_archive Lambda Output",
+  "description": "The output for the copy_to_archive Lambda.",
   "type": "object",
-  "required": ["granules", "copied_to_glacier"],
+  "required": ["granules", "copied_to_archive"],
```

Comment:
Thinking this should be copied_to_orca as this is forward facing to the user and will better define what was sent to ORCA.